arch/x86/x86.h

```diff
@@ -1,11 +1,15 @@
-/* cpu.h -- check for CPU features
+/* x86.h -- check for CPU features
 * Copyright (C) 2013 Intel Corporation Jim Kukunas
 * For conditions of distribution and use, see copyright notice in zlib.h
 */
 
 #ifndef CPU_H_
 #define CPU_H_
 
+#ifdef __cplusplus
```

Comment:
If you wrap in the benchmark code `extern "C" { }` around the include you don't need to put `#ifdef __cplusplus`

Reply:
I thought about that too but wasn't sure which way to go.

Reply:
Fixed.